.github/workflows/bump-deps.yml

```diff
@@ -1,41 +0,0 @@
-# adapted based on
```

Comment:
If we're removing this, should we also remove scripts/bump-deps.sh?

Reply:
Done